lib/travis/build/script/julia.rb

```diff
@@ -108,14 +108,17 @@ def julia_url
             end
             case julia_version = Array(config[:julia]).first.to_s
             when 'release'
-              # CHANGEME on new minor releases (once or twice a year)
-              url = "julialang-s3.julialang.org/bin/#{osarch}/0.6/julia-0.6-latest-#{ext}"
+              sh.failure "'release' is no longer a valid julia version specifier in .travis.yml," \
+                "please use '1' instead."
```

Comment:
Duh, so just listing a major is going to have the exact same problem. If you test on "1" but say you support 1.0, you won't notice if you start depending on 1.1-only features.

Reply:
This is to solve the opposite problem: ensure that seldom-updated packages aren't just being tested on a 3 year old release. We should also encourage package authors to explicitly test their minimum supported version as well (probably would be good to write somewhere).

Reply:
That had been mentioned in the attobot tagging message, wasn't it? I don't think continuing to test against a constant release is a problem at all, it's beneficial and a better default than silently changing what's running. If a package hasn't been touched, travis doesn't run nightly (unless you've asked it to). That's a different issue and not really for a package's own CI to be responsible for when what's changing between test runs is the ecosystem around it, rather than the package itself.

Reply:
So what are you saying should be changed in this PR?

Reply:
Once this is merged, my plan is that the auto-generated .travis.yml will specify to run against versions 1 and the maj.min version used to generate the package. That way we should capture both. (I may also get a script to go through the registry and do this for existing packages as well).